src/MQTTConnector.js

```diff
@@ -74,7 +74,7 @@ class MQTTConnector {
             payload_not_available: 'Offline',
             unique_id: `am43_${device.id}_battery_sensor`,
             device: deviceInfo,
-            value_template: '{{value_json[\'battery\']}}',
+            position_template: '{{value_json[\'battery\']}}',
```

Comment:
Renaming in HA was only done for cover mqtt discovery. So this one is still value template